include/ofi_util.h

```diff
@@ -547,21 +547,22 @@ util_cmap_alloc_handle(struct util_cmap *cmap, fi_addr_t fi_addr,
 		       struct util_cmap_handle **handle);
 /* Caller must hold `cmap::lock` */
 static inline struct util_cmap_handle *
+ofi_cmap_alloc_handle(struct util_cmap *cmap, fi_addr_t fi_addr)
+{
+	assert(fi_addr <= cmap->av->count);
+	struct util_cmap_handle *handle;
+
+	if (OFI_UNLIKELY(util_cmap_alloc_handle(cmap, fi_addr,
+						CMAP_IDLE, &handle)))
+		return NULL;
+	return handle;
+}
```

Comment:
Why add this simple wrapper function?

Reply:
I agree that this wrapper function isn't needed at all. Removed this